homeassistant/components/insteon/light.py

```diff
@@ -53,6 +55,9 @@ async def async_turn_on(self, **kwargs):
         """Turn light on."""
         if ATTR_BRIGHTNESS in kwargs:
             brightness = int(kwargs[ATTR_BRIGHTNESS])
+        else:
+            brightness = self.get_device_property(ON_LEVEL)
```

Comment:
The minimal brightness of 0 in Home Assistant, should reflect the minimal brightness in the device.

This needs to be remapped/rescaled to the scaling in Home Assistant is consistent across all devices.

Reply:
I don't quite understand this, sorry. As far as I can tell the ATTR_BRIGHTNESS value, the brightness argument and the ON_LEVEL raw value all go 0...255 and in testing this works flawlessly. Please explain what you're looking for. Thanks.

Reply:
Maybe the description of the change isn't clear. This only impacts the action when the user clicks the light 'On'. Using the brightness slider works as usual - so the light can be set to full bright manually if on_level is less than 255.

Reply:
Ah right, got it 👍